docs/subscribe/desktop.md

```diff
@@ -0,0 +1,12 @@
+# Using the web app as an installed PWA
```

Comment:
I suppose - just thought it would be nice to highlight that you can have an individual app in your dock/etc by using Chrome or Edge, including receiving background notifications as long as either the PWA or browser is running. There isn't much functional difference from an Electron app for an app like ntfy.

Reply:
I'd argue that the difference is the executable / installer. I very much do think there is a difference between the "Slack" I install and the Slack web app. Even though it's 99% the same. I don't care enough though. We can leave it like this, except for the "PWA" phrase in the title. I don't even know what that means, and neither does a user.

Reply:
Agreed on the wording! There is an "executable", i.e. it creates an "Application" on macOS that is completely separate from Chrome, not sure what it looks like on Linux & Windows.

In the end it all comes down to what ntfy users would want from a desktop app that a PWA couldn't provide. for e.g. I was wondering what we could do with installable PWAs – unfortunately `ntfy://` won't be possible - but registering `web+ntfy://` would 🤔 [Web/API/Navigator/registerProtocolHandler](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/registerProtocolHandler).